src/nyc_trees/nyc_trees/settings/base.py

```diff
@@ -70,6 +72,19 @@
 # END CACHE CONFIGURATION
 
 
+# CELERY CONFIGURATION
+BROKER_URL = 'redis://{0}:{1}/2'.format(
+    environ.get('NYC_TREES_CACHE_HOST', 'localhost'),
+    environ.get('NYC_TREES_CACHE_PORT', 6379))
+
+CELERY_ACCEPT_CONTENT = ['json']
+CELERY_TASK_SERIALIZER = 'json'
```

Comment:
I settled on JSON because of general recommendations around keeping data being passed around non-opaque (strings, numbers, arrays, lists, dicts). Obviously you can pass those around with Pickle too, but seemed like JSON would help enforce it.

Also, Celery throws a bunch of scary warnings if you start it without a default serializer that seem to want you to avoid Pickle:

```
Starting from version 3.2 Celery will refuse to accept pickle by default.

The pickle serializer is a security concern as it may give attackers
the ability to execute any command.  It's important to secure
your broker from unauthorized access when using pickle, so we think
that enabling pickle should require a deliberate action and not be
the default choice.

If you depend on pickle then you should set a setting to disable this
warning and to be sure that everything will continue working
when you upgrade to Celery 3.2::

    CELERY_ACCEPT_CONTENT = ['pickle', 'json', 'msgpack', 'yaml']

You must only enable the serializers that you will actually use.


  warnings.warn(CDeprecationWarning(W_PICKLE_DEPRECATED))
```